webview/src/shared/components/sectionContainer/styles.module.scss

```diff
@@ -0,0 +1,68 @@
+@import '../../variables.scss';
+@import '../../style.scss';
+
+$gap: 20px;
```

Comment:
It's used only once and it's not really a gap, I think it'll be better to just use the value or move the `$gap` variable to the `variables.scss` file.